praetorian_cli/ui/aegis/menu.py

```
@@ -139,32 +181,197 @@ def _get_schedule_completions(self, prefix):
+    now = time.monotonic()
+    if cache_key in menu._remote_ls_cache:
+        ts, entries = menu._remote_ls_cache[cache_key]
+        if now - ts < _REMOTE_LS_TTL:
+            return entries
+
+    if cache_key in menu._remote_ls_pending:
+        return None
+
+    try:
+        _, user = menu.sdk.aegis.api.get_current_user()
+    except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)